rules/c2/tool_transfer/shell.yara

```diff
@@ -46,9 +46,11 @@ rule tool_chmod_relative_run_tiny: critical {
     $o_chmod     = /chmod [\+\-\w \$\@\{\w\/\.]{0,64}/
     $o_dot_slash = /\.\/[\$a-z]{1,2}[a-z\.\/\- ]{0,32}/ fullword
 
-    $not_copyright_comment = "# Copyright"
-    $not_source            = "source ./"
-    $not_apache_license    = "Apache License"
+    $not_copyright_comment  = "# Copyright"
+    $not_source             = "source ./"
+    $not_apache_license     = "Apache License"
+    $not_ruby_health_check1 = "chruby"
+    $not_ruby_health_check2 = "RUBY_VERSION"
```

Comment:
in addition, mark this rule as `high`.

Reply:
Updated in [`4260efd` (#694)](https://github.com/chainguard-dev/malcontent/pull/694/commits/4260efda23707a4e7d4a8539902b41a090f16684).